src/KDSoapServer/KDSoapServerSocket_p.h

```diff
@@ -25,10 +25,9 @@
 
 #include <QtGlobal>
 
+#include <QTcpSocket> //krazy:exclude=includes must come first to define QT_NO_OPENSSL
```

Comment:
David is correct. you don't need to say krazy:exclude here.  the comment should be "may define QT_NO_OPENSSL"